frontend/src/components/GForceDeletion.vue

```diff
@@ -0,0 +1,185 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <g-shoot-action-dialog
+    ref="actionDialog"
+    :shoot-item="shootItem"
+    :caption="caption"
+    confirm-button-text="Delete"
+    confirm-required
+    width="600"
+  >
+    <v-list>
+      <v-list-item-subtitle>
+        Created By
+      </v-list-item-subtitle>
+      <v-list-item-title>
+        <g-account-avatar
+          :account-name="shootCreatedBy"
+          :size="22"
+        />
+      </v-list-item-title>
+    </v-list>
+    <p>
+      Type <span class="font-weight-bold">{{ shootName }}</span> below to confirm the forceful deletion of the cluster.
+    </p>
+    <p class="my-2 text-error font-weight-bold">
+      This action cannot be undone.
+    </p>
+    <g-expand-transition-group>
+      <v-alert
+        v-if="!confirmed"
+        type="warning"
+      >
+        You <span class="font-weight-bold">MUST</span> ensure that all the resources created in the IaaS account are cleaned
+        up to prevent orphaned resources. Gardener will <span class="font-weight-bold">NOT</span> delete any resources in the underlying infrastructure account.
+        Hence, use this feature at your own risk and only if you are fully aware of these consequences.
+      </v-alert>
+    </g-expand-transition-group>
+    <g-countdown-checkbox v-model="confirmed">
+      I confirm that I read the message above and deleted all resources in the underlying infrastructure account
+    </g-countdown-checkbox>
+    <p v-if="isShootReconciliationDeactivated">
+      <v-row class="fill-height">
+        <v-icon
+          color="warning"
+          class="mr-1"
+        >
+          mdi-alert-box
+        </v-icon>
+        <span>The cluster will not be deleted as long as reconciliation is deactivated.</span>
+      </v-row>
+    </p>
+  </g-shoot-action-dialog>
+  <g-shoot-action-button
```

Comment:
I'm also not sure if this can happen. And I think I also tested this... maybe only on the details page. I will add a `d-flex` div to ensure the layout does not break.